docs/cow-protocol/tutorials/cow-amm-deployer.mdx

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_position: 6
+---
+
+# CoW AMM Deployer
+
+[CoW AMM](https://cow.fi/cow-amm) is a production-ready implementation of an FM-AMM that supplies liquidity for trades made on CoW Protocol. Solvers compete with each other for the right to trade against the AMM. The winning solver is the one that moves the AMM curves higher.
+
+[CoW AMM Deployer](https://app.safe.global/share/safe-app?appUrl=https%3A%2F%2Fmonorepo-cow-amm-deployer.vercel.app&chain=eth) is an interface developed by [bleu](https://github.com/bleu-fi) to deploy and manage CoW AMMs from Safe wallets. The project was funded by the [CoW Grants DAO](https://grants.cow.fi/) to help users without programming skills to take advantage of the CoW AMM.
```

Comment:
would use the definitive domain if we plan to deploy one, otherwise fallback to  https://deploy-cow-amm.bleu.fi/ which is what we applied to get the project approved and listed by Safe